project/Dependencies.scala

```diff
@@ -106,7 +106,7 @@ object Dependencies {
   val coreSdk =
     deps ++= Seq("com.typesafe" % "config" % "1.4.2")
 
-  val devtools = deps ++= Seq(testContainers)
+  val devtools = deps ++= Seq(testContainers, scalaTest % Test)
```

Comment:
Follow-up PR will remove `testContainers`. We need it for now because of maven plugin.